sale_import_base/tests/test_sale_order_import.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright (c) Akretion 2020
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import json
+
+from odoo.exceptions import ValidationError
+
+from .common_sale_order_import import SaleImportCase
+
+
+class TestSaleOrderImport(SaleImportCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
codecov is whining about code coverage.

would it be useful to have another test_datamodel_validators.py to test pass/fail on all schema validators ?